.github/workflows/package-workflow.yml

```diff
@@ -0,0 +1,53 @@
+name: E4E Pipeline
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      PROJECT_NAME: explorer-for-endevor
+
+    strategy:
+      # Continue to run tests on the other systems if one fails
+      fail-fast: false
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: yarn install
```

Comment:
Can I ask you to specify `--frozen-lockfile` as an argument here? It will create a dependencies tree based on current lock file, instead of updating it all the time